mule-extensions-api/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>mule-extensions-api</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>Mule Extensions API</name>
+    <description>API for Mule Extensions to integrate with the Mule Runtime in a decoupled way</description>
+    <url>https://github.com/mulesoft/mule-extensions-api</url>
+
+    <parent>
```

Comment:
by code convention we define the parent first. Please follow the coding style of other poms